### PR TITLE
Make sample policy list order consistent

### PR DESCRIPTION
**Pull request by dhaiducek**
The order of the list of policies in the table didn't match the order in the links nor the order in `main.adoc`.

Also adds a description for the OPP PolicySet to the table borrowed from its documentation, with a link to its documentation (assuming it's still relevant).

ref: https://issues.redhat.com/browse/ACM-11584

### CODE REVIEW

File: governance/supported_policies.adoc [hunk 1]
```diff
@@ -54,16 +54,18 @@ Role binding policy
 
 | Gatekeeper compliance policy
 | After deploying Gatekeeper to the clusters, deploy this sample Gatekeeper policy that ensures namespaces that are created on the cluster are labeled as specified.
+
+| Red Hat OpenShift Platform Plus policy set
+| Red Hat OpenShift Platform Plus is a hybrid-cloud suite of products to build, deploy, run, and manage intelligent applications securely for multiple infrastructures. You can deploy Red Hat OpenShift Platform Plus to managed clusters using `PolicySets` delivered through a {product-title-short} application. See the link:https://access.redhat.com/documentation/en-us/openshift_platform_plus/[OpenShift Platform Plus] documentation for details on OpenShift Platform Plus.
```
Comment (dhaiducek) [inline governance/supported_policies.adoc]: @gparvin I noticed the OPP policy set wasn't in the table but had a link. I wrote up a description for it for consistency. Would you please review it?  I also found a doc link for OPP--is that relevant here?

Comment (gparvin) [reply governance/supported_policies.adoc]: Yes, this looks good Dale, thanks for the update.  Not sure how that link resolves, but hopefully it gets customers to https://access.redhat.com/documentation/en-us/openshift_platform_plus/4/html/architecture/opp-architecture

Comment (dhaiducek) [reply governance/supported_policies.adoc]: Yes, I've found it'll resolve to the top page of the latest version of the docs: https://access.redhat.com/documentation/en-us/openshift_platform_plus/

But if we don't anticipate many updates I could change it to the Architecture page directly (though it's not so many clicks to get there from the top page).

Comment (gparvin) [reply governance/supported_policies.adoc]: The top page is fine, thanks!

Comment (jc-berger) [inline governance/supported_policies.adoc]: optional: 

'intelligent' has multiple connotations, and I don't think it's needed, so consider removing. If you do want to specify a type of application, maybe say something like `advanced applications` 

Also, adverbs, like `securely` read and translate better when they immediately proceed the verbs they modify. See my suggestions below:

```suggestion
| Red Hat OpenShift Platform Plus is a hybrid-cloud suite of products to securely build, deploy, run, and manage applications for multiple infrastructures. You can deploy Red Hat OpenShift Platform Plus to managed clusters using `PolicySets` delivered through a {product-title-short} application. See the link:https://access.redhat.com/documentation/en-us/openshift_platform_plus/[OpenShift Platform Plus] documentation for details on OpenShift Platform Plus.
```

Comment (jc-berger) [inline governance/supported_policies.adoc]: Optional:

links are easier to click, and therefore help usability, if they are at the end. Consider this rewrite: 

```suggestion
| Red Hat OpenShift Platform Plus is a hybrid-cloud suite of products to build, deploy, run, and manage intelligent applications securely for multiple infrastructures. You can deploy Red Hat OpenShift Platform Plus to managed clusters using `PolicySets` delivered through a {product-title-short} application. For details on OpenShift Platform Plus, see the documentation for link:https://access.redhat.com/documentation/en-us/openshift_platform_plus/[OpenShift Platform Plus]. 
```

**Review by jc-berger — APPROVED** [review]
/lgtm

**Review by jc-berger — APPROVED** [review]
/lgtm

**Pull request merged**